cost/aws/burstable_instance_cloudwatch_credit_utilization/aws_burstable_instance_cloudwatch_credit_utilization.pt

```diff
@@ -0,0 +1,753 @@
+            "cloudwatch_cpu_surplus_credit_balance_max": result.cloudwatch_cpu_surplus_credit_balance_max,
+            "cloudwatch_cpu_surplus_credit_balance_min": result.cloudwatch_cpu_surplus_credit_balance_min,
+            "cloudwatch_cpu_surplus_credits_charged_avg": result.cloudwatch_cpu_surplus_credits_charged_avg,
+            "cloudwatch_cpu_surplus_credits_charged_min": result.cloudwatch_cpu_surplus_credits_charged_min,
+            "cloudwatch_cpu_surplus_credits_charged_max": result.cloudwatch_cpu_surplus_credits_charged_max,
+            "next_instance_size": result.instance_type_info["up"]
+          }
+        )
+      }
+    })
+  EOS
+end
+
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "pol_utilization" do
+  validate_each $ds_check_merged_metrics do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} rows containing AWS instance CloudWatch utilization data"
+    detail_template <<-EOS
+# AWS Instance Utilization
+| Region | instanceID | Instance Type | Hostname | Private DNS Name | CPU Maximum % | CPU Average % | CPU Credit Balance Minimum | CPU Surplus Credit Balance Minimum | CPU Surplus Credit Charged Maximum | Max Earnable Credits | Resize Instance To |
+| ------ | ---------- | ------------- | -------- | ---------------- | ------------- | ------------- | -------------------------- | ---------------------------------- | ---------------------------------- | -------------------- |------------------- |
+{{ range data -}}
+| {{.region}} | {{.instanceId}} | {{.instanceType}} | {{.hostname}} |  {{.privateDnsName }} | {{ .cpu_maximum }} | {{ .cpu_average }} | {{if eq .cloudwatch_cpu_credit_balance_min "10000"}}N/A{{else}}{{.cloudwatch_cpu_credit_balance_min}}{{end}} | {{if eq .cloudwatch_cpu_surplus_credit_balance_min "10000"}}N/A{{else}}{{.cloudwatch_cpu_surplus_credit_balance_min}}{{end}} | {{if eq .cloudwatch_cpu_surplus_credits_charged_max "10000"}}N/A{{else}}{{.cloudwatch_cpu_surplus_credits_charged_max}}{{end}} | {{.instance_type_info.burst_info.max_earnable_credits}} | {{.next_instance_size}} |
+{{ end -}}
+___
+###### Policy Applied in Account: {{ rs_project_name }} (Account ID: {{ rs_project_id }}) within Org: {{ rs_org_name }} (Org ID: {{ rs_org_id }})
+EOS
+    escalate $email
+    escalate $resize_instance
+    check eq(0,1)
+  end
+end
+
+
+###############################################################################
+# Escalations
+###############################################################################
+
+escalation "email" do
+  email $param_email
+end
+
+escalation "resize_instance" do
+  request_approval  do
+    label "Approve Resizing Resource"
+    description "Approve escalation to run RightScale Cloud Workflow to resize instances"
```

Comment:
Change Rightscale Cloud Workflow to action?  Users don't know about Cloud workflow.